CountryPickeriOS13+/CountryPickerSPM/Sources/CountryPickerSPM/CountryPickerWheelView/CountryPickerWheelViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by ANKUSH BHATIA on 8/2/23.
+//
+
+import Foundation
+
+final class CountryPickerWheelViewModel: ObservableObject {
+    
+    @Published var countries: [Country]
+    @Published var selected: Int
+    
```

Comment:
No need to var and published if they are not changes remove ObservableObject as well if not needed

Reply:
Done